drivers/watchdog/wdt_dw.c

```diff
@@ -21,6 +22,10 @@ LOG_MODULE_REGISTER(wdt_dw, CONFIG_WDT_LOG_LEVEL);
 struct dw_wdt_dev_data {
 	/* MMIO mapping information for watchdog register base address */
 	DEVICE_MMIO_RAM;
+#if DT_ANY_INST_HAS_PROP_STATUS_OKAY(clocks)
```

Comment:
This needs to be #if DT_ANY_INST_HAS_PROP_STATUS_OKAY(clocks). I can do #if not defined to save the memory in case of clk_freq in dw_wdt_dev_config structure

Reply:
Indeed, my mistake. I confused the data and config structures.